gravity.sh

```diff
@@ -43,7 +45,7 @@ else
 fi
 
 #Remove the /* from the end of the IPv4addr.
-IPv4addr=${IPv4_address%/*}
+IPv4_address=${IPv4_address%/*}
```

Comment:
asuming yoasuming you are refering to https://github.com/pi-hole/pi-hole/blob/promoTweaksTheSecond/automated%20install/basic-install.sh#L338